www/js/group/groupList/groupModal.html

```diff
@@ -1,8 +1,6 @@
 <ion-modal-view>
-<!--   <ion-header-bar>
-  </ion-header-bar> -->
   <ion-content>
-    <a class="button button-large button-balanced" href="#/tab/create-group">Create a new group</a>
-    <a class="button button-large button-positive" href="#/tab/join-group">Join a group</a>
+    <a class="button button-large button-balanced" ng-click='createGroup()'>Create a new group</a>
```

Comment:
The controller for this is the `NavBarCtrl` right? If so, it doesn't seem to actually be creating a group right now?